openassessment/templates/openassessmentblock/staff_area/oa_student_info.html

```diff
@@ -26,11 +26,11 @@ <h2 class="staff-info__title ui-staff__subtitle">
             {% if workflow_cancellation %}
                 <p>
                     {% if workflow_cancellation.cancelled_by %}
-                        {% blocktrans with removed_by_username=workflow_cancellation.cancelled_by removed_datetime=workflow_cancellation.cancelled_at|timezone:time_zone|date:"F j, Y H:i e" %}
+                        {% blocktrans with removed_by_username=workflow_cancellation.cancelled_by removed_datetime=workflow_cancellation.cancelled_at|date:"Y-m-d H:i" %}
                             Learner submission removed by {{ removed_by_username }} on {{ removed_datetime }}
                         {% endblocktrans %}
                     {% else %}
-                        {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|timezone:time_zone|date:"F j, Y H:i e" %}
+                        {% blocktrans with removed_datetime=workflow_cancellation.cancelled_at|date:"Y-m-d H:i" %}
                             Learner submission removed on {{ removed_datetime }}
```

Comment:
This is similar to the cancelled submission template above in the UX. Will follow up.